styles.css

```
@@ -0,0 +1,169 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	margin: 0;
+	font-family: arial, sans-serif;
+	
+}
+
+nav {
+	background-color: #019dcf;
+	color: #ffffff;
+	font-size: 15px;
+	font-weight: 400;
+	letter-spacing: .05em;
+	padding: 40px 60px;
+	
+}
+
+nav ul {
+	margin: 0;
+	width: 100%;
+	padding: 0;
+	text-align: center;
+	overflow: auto;
+}
+
+nav li {
+	float: left;
+	list-style: none;
+	width: 33%;
+}
+
+nav li:hover {
+	color: black;
+}	
+
+@media screen and (max-width: 1220px){
+	nav {
+			font-size: .9rem;
+	}
+}
+
+
+@media screen and (max-width: 500px){
+	nav {
+		width: 500px;
+		font-size: .8rem;
+	}
+}
+
+header {
+	background-color: #ffffff;
+	color: #019dcf;
+	font-size: 30px;
+	font-weight: 400;
+	letter-spacing: .05em;
+	padding: 10px 20px;
+	text-align: center;
+	overflow: auto;
+}
+
+@media screen and (max-width: 1220px){
+	header {
+		
+		font-size: .9rem;
+	}
+}
+
+@media screen and (max-width: 500px){
+	header {
+		width: 500px;
+		font-size: .8rem;
+	
+	}
+}
+
+
+.portfolio {
+	height: 400px;
+	margin: 0 auto;
+	width: auto;
+	overflow: auto;
+
+}
+
+
+
+.portfolio .project {
+	background-color: #019dcf;
+	color: #ffffff;
+	font-size: 20px;
+	font-weight: 400;
+	letter-spacing: .05em;
+	padding: 40px 60px;
+	border: 20px;
+	margin: 2px;
+	width: 33%;
+	float: left;
+	height: 400px;
+}
```

you can put most of your projectOne/Two/Three details here to save code bloat:
```suggestion
background-repeat: no-repeat;
background-size: auto;
background-position: center;
}
```